server/core/src/main/java/org/infinispan/server/core/logging/Log.java

```diff
@@ -107,4 +107,7 @@
 
 //   @Message(value = "No provider for authorization realm", id = 5037)
```

Comment:
can't be delete this comment ?

Reply:
We leave unused logs commented in there to mark ids which should no longer be used